src/main/java/com/rultor/agents/ecs/Amazon.java

```diff
@@ -50,10 +51,10 @@
 public interface Amazon {
 
     /**
-     * Run EC2 OnDemand instance.
-     * @return EC2 Instance
+     * Run ECS OnDemand instance.
+     * @return ECS Instance
      * @throws IOException if fails
      */
-    Instance runOnDemand() throws IOException;
+    ContainerInstance runOnDemand() throws IOException;
```

Comment:
@mkordas @xupyprmv it's not please see http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ecs/model/Container.html, this is what we want here, we need an actual container to run our things in, not an EC2 instance holding containers. We need to abstract this part (EC2 -> containers) away in the implementation of the interface.